kroxylicious-operator/src/test/java/io/kroxylicious/kubernetes/operator/checksum/Crc32ChecksumGeneratorTest.java

```diff
@@ -123,6 +111,58 @@ void shouldIncludeGenerationInChecksum() {
                 .isNotEqualTo(proxyChecksum);
     }
 
+    @Test
+    void shouldAppendGenerationToChecksum() {
+        // Given
+        String proxyChecksum = MetadataChecksumGenerator.checksumFor(List.of(PROXY));
+        Crc32ChecksumGenerator generator = new Crc32ChecksumGenerator();
+        generator.appendMetadata(PROXY);
+
+        // When
+        String checksum = generator.encode();
+
+        // Then
+        assertThat(checksum)
+                .isNotBlank()
+                .isEqualTo(proxyChecksum);
+    }
+
+    @Test
+    void shouldGenerateDifferentChecksumsForRepeatedCallsToAppendLong() {
+        // Given
+        String filterUid = UUID.randomUUID().toString();
```

Comment:
Is this used? I guess we could remove it for this and the next test

Reply:
Probably not actually needed. The test went through a few iterations while we found the real problem